src/components/gocam-legend/gocam-legend.scss

```diff
@@ -28,6 +28,7 @@
 
 .header {
   @include standard-var-styles(header);
+  text-align: center;
```

Comment:
I know it seems like a minor thing, but I'd prefer to keep the base style as unopinionated as possible. If a client wants this text centered there is already a way for them to accomplish that, as illustrated here: https://github.com/geneontology/wc-gocam-viz/blob/5c373a9e46f77b8b623f7bab90670906987c4c60/src/styled.html#L46-L48